tensorflow_tts/processor/base_processor.py

```diff
@@ -193,4 +199,6 @@ def _save_mapper(self, saved_path: str = None):
                 "id_to_symbol": self.id_to_symbol,
                 "speakers_map": self.speakers_map,
             }
+            if extra_attrs_to_save:
```

Comment:
@machineko let take a look if it make sense :)).

Reply:
LGFM